cli/flox/src/commands/build.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Result;
+use bpaf::Bpaf;
+use flox_rust_sdk::flox::Flox;
+use tracing::instrument;
+
+use super::{environment_select, EnvironmentSelect};
+use crate::config::Config;
+use crate::subcommand_metric;
+use crate::utils::message;
+
+#[allow(unused)] // remove when we implement the command
+#[derive(Bpaf, Clone)]
+pub struct Build {
+    #[bpaf(external(environment_select), fallback(Default::default()))]
+    environment: EnvironmentSelect,
+
+    /// Whether to print logs to stderr during build
+    /// Logs are always written to <TBD>
```

Comment:
The continuation in the output looked odd:
```
    -L, --build-logs  Whether to print logs to stderr during build Logs are
                      always written to <TBD>
```
```suggestion
    /// Whether to print logs to stderr during build.
    /// Logs are always written to <TBD>
```